src/SFML/System/Android/SuspendAwareClock.cpp

```diff
@@ -22,31 +22,24 @@
 //
 ////////////////////////////////////////////////////////////
 
-#ifndef SFML_SLEEPIMPLUNIX_HPP
-#define SFML_SLEEPIMPLUNIX_HPP
-
 ////////////////////////////////////////////////////////////
 // Headers
 ////////////////////////////////////////////////////////////
-#include <SFML/Config.hpp>
-#include <SFML/System/Time.hpp>
+#include <SFML/System/Android/SuspendAwareClock.hpp>
+#include <ctime>
 
+#ifdef CLOCK_BOOTTIME
 
-namespace sf
-{
-namespace priv
+namespace sf::priv
 {
-////////////////////////////////////////////////////////////
-/// \brief Unix implementation of sf::Sleep
-///
-/// \param time Time to sleep
-///
-////////////////////////////////////////////////////////////
-void sleepImpl(Time time);
-
-} // namespace priv
 
-} // namespace sf
+SuspendAwareClock::time_point SuspendAwareClock::now() noexcept
+{
+    timespec ts;
+    clock_gettime(CLOCK_BOOTTIME, &ts);
```

Comment:
Qualify those two with `std::` (they're part of `<ctime>`, which _officially_ is part of `std` in contrast to `<time.h>`).

Reply:
Not possible. `timespec` and `clock_gettime` are not part of the Standard. Those are Linux-specific.

I chose `<ctime>` because the header existence is guaranteed by the Standard (e.g. older OSX would include `mach_time.h`).

Reply:
C++17 added [`timespec`](https://en.cppreference.com/w/cpp/chrono/c/timespec), but it looks like `clock_gettime()` is really POSIX.

In that case, it's fine if you leave them out of the namespace 👍

Reply:
Oh, my bad then. I see an error squiggle at `std::timespec` but it compiles. Very weird.

Reply:
It _probably_ works because the `std` symbol is simply pulled in with `using` from the global one, but maybe it's safer to keep both in the global namespace. You can qualify it with `::timespec` to be explicit...

Reply:
It was because I installed clang-tidy which pulled Clang-13 into PATH. This must be a clang-13 bug with M1. Now, it's alright

Reply:
Android fails too. Compilers supporting C++17 but lacking `std::timespec`. I'll revert to global namespace.